filterBOM.js

```diff
@@ -22,8 +23,10 @@ async function run() {
   const outfile = process.argv[3] || infile;
 
   const data = require(infile);
+  console.log('Before: ' + data.components.length + ' components.');
   data.components = uniqueComponents(data.components);
+  console.log(' After: ' + data.components.length + ' components.');
```

Comment:
Are these console.logs intentional?

Reply:
yes. Expected output for this script is something like:

```
Generating /Users/erichs/repos/jupiterone/node-cdx-bom/bom.json...
Finding packages...
Generating BOM for ./package.json...

Merging BOM file(s)...
Processing input file /Users/erichs/repos/jupiterone/node-cdx-bom/bom.xml
    Contains 46 components
Writing output file...
    Total 46 components

Cleanup intermediate files...

Deduping BOM file...
Before: 46 components.
 After: 46 components.

Analyzing BOM...
Analysis results:

Components with multiple versions:

None


Validating BOM...
Validating JSON SBOM...
Generated BOM file /Users/erichs/repos/jupiterone/node-cdx-bom/bom.json - OK
```